pennylane_qulacs/qulacs_device.py

```diff
@@ -295,12 +295,44 @@ def analytic_probability(self, wires=None):
         prob = self.marginal_prob(all_probs, wires)
         return prob
 
+    def expval(self, observable):
+        observable_map = {
+            "PauliX": "X",
+            "PauliY": "Y",
+            "PauliZ": "Z",
+            "Identity": "I",
+            "Hadamard": None,
+            "Hermitian": None,
+        }
```

Comment:
Should we not put this in the header to make it global? Or is there a reason to redefine it in every expval call?